app/src/main/java/org/simple/clinic/storage/monitoring/DbMethodLookup.kt

```diff
@@ -0,0 +1,46 @@
+package org.simple.clinic.storage.monitoring
+
+import androidx.annotation.VisibleForTesting
+import androidx.annotation.VisibleForTesting.PRIVATE
+
+class DbMethodLookup @VisibleForTesting(otherwise = PRIVATE) constructor(
```

Comment:
How do we create an instance of this class outside of the tests? Do we use a factory method?

Reply:
NVM, resolved in next commit.